### PR TITLE
Removing unwanted tabs from the UI.

### PR DESCRIPTION
**Pull request by joshirohit100**

### CODE REVIEW

**Comment by joshirohit100** [comment]
now available for review. @amitgoyal @pranitjha @gulabbisht

**Comment by joshirohit100** [comment]
screen shot 2019-01-15 at 7 03 42 pm

File: migration_glue.links.menu.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 migration_glue.create_migration:
-  title: 'Migration Glue'
+  title: 'Acquia plateform migration'
```
Comment (amitgoyal) [inline migration_glue.links.menu.yml]: Spelling mistake :)
It should be "Acquia Platform Migration".

Comment (joshirohit100) [reply migration_glue.links.menu.yml]: PR updated! Thanks!

**Pull request merged**